scripts/exregional_run_met_pb2nc_obs.sh

```diff
@@ -233,46 +233,46 @@ settings="\
 #
 # MET/METplus information.
 #
-  'metplus_tool_name': '${metplus_tool_name}'
```

Comment:
Is the removal of indentation here a new requirement or a style choice?

Reply:
No, it ended up being a red-herring during some debugging. I will put it back as it was.